src/routes/userRoutes.js

```diff
@@ -81,6 +81,31 @@ routes.deleteUser = async (req, res) => {
       res.status(500).json({error: {message: 'Deleting user failed. Please try again'}})
   }
 }
+  routes.updateDBPassword = async(req,res) =>{
+    if(req.params.id===null ||!req.body.password ){
+      logger.info('invalid input')
+      return res.status(400).json({error:{message:'invalid input of userid and password'}})
+    }
+    const {Accounts} = db.getModels()
+    const userAccount = await Accounts.findOne({
+        where:{
+          id:req.params.id
+        }
+    })
+    if(!userAccount){
+        logger.info(`account does not exist`)
+        res.status(400).json({error:{message:"account does not exist"}})
+        return
+    }
+    try {
+      await setDBPassword(userAccount,req.body.password) 
+      logger.info(`user ${userAccount.id} updates password`)
+      return res.status(200).json('success')
```

Comment:
`success` -> `{success: {message: 'success'}}`

Reply:
I don’t think it’s necessary to complicate the message. Success is enough; at least I see most of the success messages in js5 class file is just ‘success’ in json . However error message usually tell you why it threw error.

Reply:
You're right. Send actual data instead

Reply:
@AliceHuang1027 Could you return updated user object instead of a 'success' string? Here's an example: https://github.com/garageScript/databases/pull/89/files

Reply:
Yea will fix it too